### PR TITLE
Fix size display for hm.com

### PR DESCRIPTION
**Pull request by ModProg**
Before: 

grafik

After:
grafik

Source: https://www2.hm.com/de_de/productpage.1072556002.html

### CODE REVIEW

**Comment by ModProg** [comment]
Fix rating as well:
Before:
grafik

After:
grafik

Source: https://www2.hm.com/de_de/productpage.1095628002.html

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -9083,6 +9083,18 @@ body
 
 ================================
 
+hm.com
+
+CSS
+.review-answers .review-answer .average-score {
+    background-color: var(--darkreader-neutral-text) !important;
+}
+.TurnToReviewsButton-module--ratingStars__gFrXo {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9092]: What about using:

```css
INVERT 
[class="TurnToReviewsButton-module--ratingStars"]
```
?

Comment (ModProg) [reply src/config/dynamic-theme-fixes.config:9092]: I didn't know that was a thing, will correct it

Comment (ModProg) [reply src/config/dynamic-theme-fixes.config]: fixed

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -9083,6 +9083,18 @@ body
 
 ================================
 
+hm.com
+
+INVERT
+[class="TurnToReviewsButton-module--ratingStars__gFrXo"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9089]:
```suggestion
[class^="TurnToReviewsButton-module--ratingStars"]
```

**Review by Gusted — COMMENTED** [review]
Small nit, otherwise LGTM!

**Review by Gusted — APPROVED** [review]

**Comment by Gusted** [comment]
Thanks!

**Comment by wan7teen-ping** [comment]
Hi I will help you with everything you need but you have to subscribe to
the youtube channel and I will fulfill all your requests

https://youtu.be/aIaqr4eq18g

Pada Sab, 27 Ogo 2022, 3:42 PM Roland Fredenhagen ***@***.***>
menulis:

**Pull request merged**